src/main/java/io/codelex/enums/practice/Exercise2/ScissorPaperStone.java

```diff
@@ -0,0 +1,22 @@
+package io.codelex.enums.practice.Exercise2;
+
+public enum ScissorPaperStone {
+    SCISSOR,
+    PAPER,
+    STONE;
+
+    public String toString() {
```

Comment:
šķiet jau ka built in metode pārveidošanai uz string ar pilnībā derētu, ja nu vienīgi tiesi gribēji savādāk 😄

Reply:
reti, kad biju lietojis switch statment.
Treniņa nolūkos šeit ievietoju SWITCH statement.
Šādi tu domā? 
  SCISSOR("SCISSOR"),
    PAPER("PAPER"),
    STONE("STONE");
    
    String stringName;

    ScissorPaperStone(String stringName) {
        this.stringName = stringName;
    }

    @Override
    public String toString() {
        return "ScissorPaperStone{" +
                "stringName='" + stringName + '\'' +
                '}';
    }

Reply:
nē nē, its like 
`SCISSOR.toString()` vai `SCISSOR.name()`